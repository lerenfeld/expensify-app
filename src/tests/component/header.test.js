import React from 'react'
import Header from '../../components/Header'
import {shallow} from 'enzyme'

test('should render Header correctly',  ()=>{
  const wrapper = shallow(<Header/>)

  // const renderer = new ReactshallowRenderer 
  // renderer.render(<Header />)
  // expect(renderer.getRenderOutput()).toMatchSnapshot()

  // expect(wrapper.find('h1').text()).toBe('Expensify');

  expect(wrapper).toMatchSnapshot();
});



