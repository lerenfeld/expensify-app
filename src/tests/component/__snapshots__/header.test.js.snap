// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <[object Object]
    className={[Function]}
    to="/"
  >
    Home
  </[object Object]>
  <[object Object]
    className={[Function]}
    to="/create"
  >
    create
  </[object Object]>
  <[object Object]
    className={[Function]}
    to="/help"
  >
    help
  </[object Object]>
</header>
`;
